packages/apps-config/src/endpoints/testingRococo.ts

```diff
@@ -263,7 +263,7 @@ export function createRococo (t: TFunction): LinkOption[] {
         // Needs paraId >= 1000, below that is reserved for system parachains
         // Update paraId to new selection, remove this flag to enable
         isDisabled: true,
-        paraId: 777,
+        paraId: 7777,
```

Comment:
Just remove the isSisabled above the paraId as well. (Otherwise it does not show up)

Reply:
Thanks! Fixed.